.github/workflows/push-deploy.yml

```diff
@@ -0,0 +1,16 @@
+name: Deploy to WordPress.org
+on:
+  push:
+    tags:
+    - "*"
+jobs:
+  tag:
+    name: New tag
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: WordPress Plugin Deploy
+      uses: 10up/action-wordpress-plugin-deploy@develop
+      env:
```

Comment:
@helen does this also need the `GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}` bit?

Reply:
No it shouldn't, I changed it so it shouldn't need that anymore.